modules/post/windows/escalate/getsystem.rb

```diff
@@ -28,13 +30,21 @@ def initialize(info = {})
               priv_elevate_getsystem
             ]
           }
+        },
+        'Actions' => techniques.map.with_index {|t,i| [i.to_s, { 'Description' => t }]},
+        'DefaultAction' => '0',
+        'Notes' => {
+          'AKA' => [
+            'Named Pipe Impersonation',
+            'Token Duplication',
+            'RPCSS',
+            'PrintSpooler',
+            'EFSRPC',
+            'EfsPotato'
+          ]
```

Comment:
Since the `ACTIONS` name and description are not searchable with `search` command, I added some keywords as `AKA`. This might change in the future.

Reply:
> This might change in the future

https://github.com/rapid7/metasploit-framework/pull/18581 🤞